Vungle/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     if (sdkSrcDependency) {
         implementation project(':vungle-android-sdk:publisher-sdk-android')
     } else {
-        implementation "com.vungle:publisher-sdk-android:${sdkVersion}"
+        //implementation "com.vungle:publisher-sdk-android:${sdkVersion}"
+        implementation(name:'vungle-android-sdk-6.10.0-RC1', ext: 'aar')
```

Comment:
Should it be on maven?

Reply:
No, build failed due to fat-aar: https://jitpack.io/com/github/Vungle/vungle-android-sdk/6.10.0-RC1/build.log

Reply:
I mean Mopub will expect it on Maven, not .aar

Reply:
yes, considering SDK will be published on maven then when submitting official PR.